tests/test_sft_trainer.py

```diff
@@ -63,7 +63,6 @@
     "save_strategy": "epoch",
```

Comment:
just a more general question - is there any scenario you can imagine where `{{foo}}` literal is actually expected to be part of the template, and not something to render, as this would currently not be allowed?

Reply:
ya we ll make it clear that its not allowed. Only keys of template should be in {{}} , if anyone has other requirements they can raise request